pkgs/games/duckmarines/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, unzip, love, lua, makeWrapper, makeDesktopItem }:
+
+let
+  pname = "duckmarines";
+  version = "1.0b";
+
+  icon = fetchurl {
+    url = "http://tangramgames.dk/img/thumb/duckmarines.png";
+    sha256 = "07ypbwqcgqc5f117yxy9icix76wlybp1cmykc8f3ivdps66hl0k5";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = "duckmarines";
+    exec = "${pname}";
+    icon = "${icon}";
+    comment = "Duck-themed action puzzle video game";
+    desktopName = "Duck Marines";
+    genericName = "duckmarines";
+    categories = "Game;";
+  };
+
+in
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/SimonLarsen/${pname}/releases/download/v${version}/${pname}-1.0-love.zip";
+    sha256 = "0fpzbsgrhbwm1lff9gyzh6c9jigw328ngddvrj5w7qmjcm2lv6lv";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ lua love ];
+
+  phases = "unpackPhase installPhase";
```

Comment:
`phases` expects a single string if there is only a single phase. If there are more than one you have to define a list of strings. Therefore `phases = [ "unpackPhase" "installPhase" ]` would be correct.